src/ServiceManagement/Automation/Commands.Automation/Cmdlet/SetAzureAutomationVariable.cs

```diff
@@ -57,11 +66,20 @@ protected override void AutomationExecuteCmdlet()
             {
                 Name = this.Name,
                 Description = this.Description,
-                Value = this.Value
+                Encrypted = this.Encrypted,
+                Value = JsonConvert.SerializeObject(this.Value),
```

Comment:
since we are already serializing the value in AutomationClient, why do we need to serialize it here as well?